addons/web_editor/static/src/js/wysiwyg/widgets/image_crop.js

```diff
@@ -151,6 +154,9 @@ export class ImageCrop extends Component {
         this.$cropperImage = this.$('.o_we_cropper_img');
         const cropperImage = this.$cropperImage[0];
         [cropperImage.style.width, cropperImage.style.height] = [this.$media.width() + 'px', this.$media.height() + 'px'];
+        
+        const sel = this.wysiwyg.odooEditor && this.wysiwyg.odooEditor.document.getSelection();
```

Comment:
I don't think you need to add `wysiwyg` as a prop just to reach the document. Assuming `this.media` is the image we are editing, it _is_ in the document. Therefore, you should be able to do `this.media.ownerDocument` to get the document of the media.